static/css/base.css

```diff
@@ -148,13 +149,13 @@ body {
 
 .page-checks .container-fluid,
 .page-details .container-fluid {
-    /* Fluid below 1320px, but max width capped to 1320px ... */
-    max-width: 1320px;
+    /* Fluid below 1170px, but max width capped to 1170px ... */
+    max-width: 1170px;
```

Comment:
I understand that but then why don't put all the pages in this width instead of changing between each pages that make transitions look weird

Reply:
IIRC I started with Bootstrap's default width for all pages. I needed to make the pages wider in a few places to make the content fit better, but didn't want to make the change globally, because then I would need to review all other pages and fix instances where the  wider layout looks bad (for example, because there's not much content on the page). So I made selective changes in just a few pages.